src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -350,7 +362,30 @@ private function autocomplete(OutputInterface $output, Question $question, $inpu
         // Reset stty so it behaves normally again
         shell_exec(sprintf('stty %s', $sttyMode));
 
-        return $ret;
+        return $fullChoice;
+    }
+
+    /**
+     * Determine the most recent value the user entered.
+     *
+     * @param $entered
+     *
+     * @return string
+     */
```

Comment:
Can you remove the entire docblock? It does not have much value

Reply:
I've removed the docblock now